### ISSUE TITLE
Datenbank Verbindung

### ISSUE THREAD

**Issue by JonasLiendl** [misterorora/to-do-list#12]
Damit die erstellten Aufgaben, Projekte und Labels nicht verloren gehen müssen diese gespeichert werden. Einigung ist dies mit SQLite zu speichern. Somit ist eine Klasse benötigt, welche die Verbindung mit der Datenbank und Datenabfragen sowie Anfragen zur Speicherung von Daten verwaltet.
Dabei soll es Methoden für die Interaktion mit Benutzern, Aufgaben, Projekten und Labels geben.

**Inhalt**

- [x] Eine Klasse die mit der Datenbank kommuniziert erstellen
- [ ] Bei Initialisierung durch "main" den Dateipfad und die Datenbankdatei anlegen
- [ ] Interfaces für Datenabfrage und Datenspeicherung erstellen (lose Kopplung)
- [ ] Tests für das Verbinden zur Datenbank
- [ ] Tests für Abfragen
- [ ] Tests für Speicherungen

**Comment by misterorora** [comment]
Fälschlicherweise geschlossen

**Comment by JonasLiendl** [comment]
Habe gerade beim Durchführen der Tests einen Fehler beim Verbinden zur Datenbank bekommen. Kann es sein, dass die env Variablen nicht direkt von IntellijIDEA erkannt werden?

<img width="820" alt="Screenshot 2023-05-13 at 11 55 06" src="https://github.com/misterorora/to-do-list/assets/57270170/63924fe2-af0a-4994-bc9c-26b81bd073e6">

**Comment by misterorora** [comment]
> Habe gerade beim Durchführen der Tests einen Fehler beim Verbinden zur Datenbank bekommen. Kann es sein, dass die env Variablen nicht direkt von IntellijIDEA erkannt werden?
> <img alt="Screenshot 2023-05-13 at 11 55 06" width="820" src="https://user-images.githubusercontent.com/57270170/238127589-63924fe2-af0a-4994-bc9c-26b81bd073e6.png">

Ja, da hast du leider Recht, das ist mir auch schon aufgefallen. 
Wollte ich im nächsten Patch verändern und globale Variablen definieren dafür.

Habe allerdings bereits in die "Run_All_Tests.xml", die im Run Verzeichnis sind, die globalen Variablen eingebunden. Startest du es von dort, laufen alle Tests ohne Probleme.
